### DISCUSSION TITLE
Supporting multiple users and git all at once

### DISCUSSION THREAD

**Discussion by cpopell** [st3v3nmw/obsidian-spaced-repetition#207] [General]
So, I've been a big evangelist for this plugin - without a doubt, it's one of the single most important pieces of software I've ever used ( https://Ko-fi.com/home/coffeeshop?txid=41cafe67-2a68-4250-9f2f-290d435d67ae&mode=public&img=ogiboughtsomeone ).

I was talking with friends about it, and we were brainstorming how to use it for multiple people (i.e. if a company had a standardized obsidian vault) and for Git. The idea we had was switching from the current SR html comment to one that generates a unique identifier, that points at a metadata file where all that's stored.

This means that:

* Only one file gets edited whenever you do spaced repetition, aside from adding new cards.
* If you put it into your gitignore, people can use the same deck.

Anyway, the kofi is because you deserve it. This isn't a 'feature request' so much as thoughts about the problem.

**Comment by st3v3nmw** [comment]
Hey, first off, thanks for your support!
The current implementation (storing the scheduling in the md file) has been the main problem blocking the implementation of the most of the things on the issues list. I've been considering moving to a dedicated file as you've mentioned but I'm yet to figure out how one would handle reviewing on different devices, syncing everything together and properly handling conflicts in such as huge file.
But yes, this should be the way forward.
Thanks for your input!
